docs/4-recommended-coding-set.md

```diff
@@ -0,0 +1,132 @@
+# Recommended Coding Set
+
+## Type of Test ([Loinc](http://loinc.org))
+
+> **FHIR Mapping**: Observation.code.coding[0].{ system, code, display }
+
+<table>
+<thead>
+  <tr>
+    <th>Type</th>
+    <th>Code</th>
+    <th>Display</th>
+    <th>EU DCC</th>
+    <th>GPay COVID Card</th>
+  </tr>
+</thead>
+<tbody>
+  <tr>
+    <td rowspan="2">"PCR"</td>
+    <td>94531-1</td>
+    <td>SARS-CoV-2 (COVID-19) RNA panel - Respiratory specimen by NAA with probe detection</td>
```

Comment:
it is possible to change display name as `SARS-CoV-2 (COVID-19) RNA [Presence] in Specimen by NAA with probe detection` also ? Current EU schema not allow test name `nm` not more than 80 chars.

Reply:
The display field shown here is for following the LOINC long common name: https://loinc.org/94531-1/

I thought [EU schema has its own display naming](https://github.com/ehn-dcc-development/ehn-dcc-schema/blob/release/1.3.1/valuesets/test-type.json)?

```json
{
  "LP6464-4": {
    "display": "Nucleic acid amplification with probe detection",
    "lang": "en",
    "active": true,
    "version": "2.69",
    "system": "http://loinc.org"
  },
  "LP217198-3": {
    "display": "Rapid immunoassay",
    "lang": "en",
    "active": true,
    "version": "2.69",
    "system": "http://loinc.org"
  }
}
```

In any case, we don't validate the display field at this moment. Collaborators can always specify any display field of any length. Suggest that the EU DCC generation just trim if > 80 characters?

Reply:
yea.. can trim in eu dcc generator , but don't know where should cut off name like last part or something. And for PCR, eu dcc accept extra test name string along with PCR test type code  `LP6464-4`